package.json

```diff
@@ -39,6 +39,7 @@
     "@vercel/node": "1.12.1",
     "ajv": "8.11.0",
     "aws-sdk": "2.1053.0",
+    "azure-storage": "^2.10.7",
```

Comment:
This package is deprecated. https://www.npmjs.com/package/azure-storage

Can we replace it with https://github.com/svnlto/azure-blob-store to conform to other storage plugins?

Reply:
The package you suggested has only 1 star & last commit is from 2017.  
The only one i've found to be compatible with `abstract-blob-store` API is this one unfortunately.

Reply:
@izi-p , great work, but why not use latest SDK: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/storage/MigrationGuide.md ?

Reply:
I'm not an expert but it seems that it doesn't implement the abstract-blob-store API. there is no function like creadeReadStream or CreateWriteStream. It might need some extra-work to plug it properly with the new azure sdk

Reply:
I don't even know if it's possible as the new Azure SDK provides only async upload and download methods. Here we have to implement synchronous "createReadStream" and "createWriteStream" interface

Reply:
@fox1t sure, looks promising indeed :)

Reply:
the only solution i've found is doing something like this 
![image](https://user-images.githubusercontent.com/10976962/216389783-0134b864-a922-4adb-9a6b-24df12287c7a.png)

![image](https://user-images.githubusercontent.com/10976962/216389834-dcdd2d11-1c70-44aa-9dd8-fd24a8bb313e.png)

But it means touching the core logic of the repo, it's not only adding a provider.

Reply:
That's why I considered it was not a bad arbitrage to use the deprecated but still updated old azure package

Reply:
I never thought it was a lousy arbitrage :)

Let's use that old one for the moment, then. We are working on https://github.com/ducktors/storagebus and our plan is to replace the storage provider part to support even more providers more easily.